tv.plex.Plexamp.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: tv.plex.Plexamp
+
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+
+command: startplexamp
+separate-locales: false
+
+finish-args:
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=org.mpris.MediaPlayer2.Plexamp
+
+modules:
+  - name: assets
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 assets/startplexamp /app/bin/startplexamp
+      - install -D tv.plex.Plexamp.metainfo.xml -t /app/share/metainfo/
+      - install -D assets/tv.plex.Plexamp.desktop -t /app/share/applications/
```

Comment:
the appimage come without a .desktop or icon?

Reply:
The appimage's .desktop is quite different to our flatpak, and while it provides an icon, I wasn't able to figure out how to correctly pull it from the unzipped bundle and drop it where it needed to go.

Down the line we hope to drop appimage completely and just ship a flatpak bundle directly to Flathub, so I didn't want to invest too much time in this, given it's likely only going to stay this way for a few months or so.

Reply:
nothing that `desktop-file-edit` can't fix.